### ISSUE TITLE
Review asked for the 503 in Maintenance mode

### ISSUE THREAD

**Issue by axyr** [axyr/silverstripe-console#11]
Right now the 503 is thrown in a ControllerExtension::onBeforeInit();

https://github.com/axyr/silverstripe-console/blob/master/code/Extensions/MaintenanceModeExtension.php

This will probably not work if there is a database or missing file error during deployment.

Should I rewrite the .htaccess so all requests wont even hit the application when in Maintenance mode?

**Comment by robbieaverill** [comment]
> Should I rewrite the .htaccess so all requests wont even hit the application when in Maintenance mode?

Yes, absolutely! This way the application can be as unstable as required and always serve a static message on the front.
